manifests/4.6/cluster-logging.v4.6.0.clusterserviceversion.yaml

```diff
@@ -315,6 +315,11 @@ spec:
                 imagePullPolicy: IfNotPresent
                 command:
                 - cluster-logging-operator
+                ports:
+                  - containerPort: 8383
+                    name: http-metrics
+                  - containerPort: 8686
+                    name: cr-metrics
```

Comment:
Have you tested this on a test cluster? Do the SVCs use the ports by name or by port-number directly? If by name, do the names match between here and SVCs?

Reply:
Yes, i tested it. All deployed normally. Service use port-number directly.